src/main/webapp/scripts/map.js

```diff
@@ -199,9 +199,9 @@ function placeMarker(markerDetails, map) {
   marker = new google.maps.Marker ({
```

Comment:
Please declare it as a `const` here too ;)

Reply:
Thanks!